### DISCUSSION TITLE
Close Particle Display

### DISCUSSION THREAD

**Discussion by 123456-png** [bezo97/IFSRenderer#43] [Q&A] [answered]
Hi, is there any way to set the value at which the particles are still rendered? I would like to take a photo from inside the nebula, but it almost completely disappears at this close. I hope it’s written clearly, if so it’s Google translator

**Comment by bezo97** [comment, marked as answer]
Hi, question is clear. I see two possible ways to do this.  
First, the reason particles disappear is because there are too many of them, but this is good. What you want to achieve actually, is to shape them into dense areas so they form big particles. In the [Nebula tutorial](https://github.com/bezo97/IFSRenderer/wiki/Starfield-with-Nebula) it is the `Gaussian blur` transform that creates this effect. You can try to put more gaussian blurs and connect it to other transforms and I think that will create more stars.
Second, if the previous method doesn't work you can try to lower the amount of particles (but this will result in a noisy image). To do this you can go to `Settings`/`Performance` and set the `Workgroups` to very small. Also, setting the `Syntropy` value very high might help too.
